Updates Python Workers examples to use new Python SDK API (#18596)
cloudflare/cloudflare-docs: src/content/docs/workers/examples/aggregate-requests.mdx

</TabItem> <TabItem label="Python" icon="seti:python">

```py
from cloudflare.workers import Response, fetch

> import asyncio

Author: Yeah

import json

async def on_fetch(request):
    # some_host is set up to return JSON responses
    some_host = "https://jsonplaceholder.typicode.com"
    url1 = some_host + "/todos/1"
    url2 = some_host + "/todos/2"

>     responses = await asyncio.gather(*[fetch(url1), fetch(url2)])

Reviewer: ```suggestion
    responses = await asyncio.gather(fetch(url1), fetch(url2))
```

>     results = await asyncio.gather(*map(lambda r: r.json(), responses))

Reviewer: ```suggestion
    results = await asyncio.gather(*(r.json() for r in responses))
```

    headers = {"content-type": "application/json;charset=UTF-8"}

>     return Response(json.dumps(results), headers=headers)

Reviewer: It would nice to add a `Response.json()` method. That could also set the content type.

Author: we do have this

Reviewer: Should we use it here?
